### ISSUE TITLE
Swagger UI extension not working properly on Windows

### ISSUE THREAD

**Issue by cemnura** [quarkusio/quarkus#3313]
**Describe the bug**
swagger-ui does not work in development mode or quarkus-quickstarts\using-openapi-swaggerui test on Windows OS. 

**Expected behavior**
**quarkus-quickstarts\using-openapi-swaggerui**
**./mvnw test**

Run Success

**Actual behavior**
**quarkus-quickstarts\using-openapi-swaggerui**
**./mvnw test**

[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.001 s <<< FAILURE! - in org.acme.openapi.swaggerui.SwaggerUiTest
[ERROR] testSwaggerUi  Time elapsed: 0.001 s  <<< FAILURE!
java.lang.AssertionError:
1 expectation failed.
Expected status code <200> but was <404>.

**To Reproduce**
1. Open the quarkus-quickstarts\using-openapi-swaggerui on Windows OS
2. Run **./mvnw test**

**Environment**
Windows 10 Enterprise Version 1809
java version "11.0.1" 2018-10-16 LTS

**Configuration**
```properties
# Configuration file
# key = value
```

**Environment (please complete the following information):**
- Microsoft Windows [Version 10.0.17763.557]
- java version "11.0.1" 2018-10-16 LTS
  Java(TM) SE Runtime Environment 18.9 (build 11.0.1+13-LTS)
  Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 - Quarkus version 0.19.0 & 0.19.1

**Comment by gsmet** [comment]
Could someone having the issue on Windows try to add a breakpoint here: https://github.com/quarkusio/quarkus/blob/master/extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java#L154

Then run `mvn clean compile quarkus:dev -Ddebug` and see if the jarEntry path has `\` in it or `/` (please copy/paste the path here).

Thanks.

**Comment by lukaszkostrzewa** [comment]
From my investigation it seems that there is a problem releated to Windows short paths (the one with `~`) in `undertow-core` in a method `PathResourceManager::isFileSameCase`.

`normalizeFile` is equal to:
```
C:\Users\myusern~1\AppData\Local\Temp\quarkus-swagger-ui_68267708355431116007045174363617492
```
whereas `canonicalName` is equal to:
```
C:\Users\myusername\AppData\Local\Temp\quarkus-swagger-ui_68267708355431116007045174363617492
```
and hence it returns `false` and doesn't return a resource.

**Comment by gwenneg** [comment]
Fixed in #3451
